### DISCUSSION TITLE
Why is not-found page triggered even if the page exists?

### DISCUSSION THREAD

**Discussion by mastoj** [vercel/next.js#55583] [App Router]
I was surprised to notice that the not-found page was "triggered" even though I hit an existing url. I've created a minimal app showing what I mean: https://github.com/mastoj/nextjs-notfound

If you visit "/hello/whatever" that page should exists, but if you look in the server console you will see `Found the not found` printed which is only logged in the not-found page, https://github.com/mastoj/nextjs-notfound/blob/main/app/not-found.tsx.

Why is the not-found.tsx run even though the url exists?

**Comment by One041** [comment]
Still no any answer?
